data/base-spells.yaml

```diff
@@ -149,81 +149,81 @@ khri_preps:
 
 barb_abilities:
   Python:
-    activate: form python
-    start: imitating the cunning ways of the python
-    expire: finish practicing the Form of the Python
+    start_cmd: form python
```

Comment:
I think cmd and msg are pretty universally known? I can add in the full message, it's not really that big of a deal and I definitely agree with you for the majority of abbreviations, but these seem pretty standard.